python/paddle/fluid/layers/nn.py

```diff
@@ -4892,15 +4892,22 @@ def reshape(x, shape, actual_shape=None, act=None, inplace=True, name=None):
                                 :attr:`shape` specifying shape. That is to
                                 say :attr:`actual_shape` has a higher priority
                                 than :attr:`shape`.
-        act (str): The non-linear activation to be applied to output variable.
-        inplace(bool): If this flag is set true, the output
-                       shares data with input without copying, otherwise
-                       a new output tensor is created
-                       whose data is copied from input x.
+        act (str): The non-linear activation to be applied to the reshaped tensor
+                   variable.
+        inplace(bool): If this flag is set true, reuse input :attr:`x` to reshape,
+                       which will change the shape of tensor variable :attr:`x`.
+                       Otherwise, preserve the shape :attr:`x` and create a new
+                       output tensor variable whose data is copied from input x
+                       but reshaped. Though setting to :attr:`True` will be more
+                       efficient, :attr:`False` is suggested when :attr:`x` are
```

Comment:
what does `suggested` mean? Does it mean the result can be wrong when `x` is used in multiple operators? If true, you should be very clear in the doc.

Reply:
Done. Clarify the danger in the usage of `inplace`.